quactography/visu/dist_prob_edge.py

```diff
@@ -37,13 +39,31 @@ def plot_distribution_of_probabilities_edge(
     dist_binary_prob = dist_binary_prob.item()
     min_cost = min_cost.item()
     h = h.item()
+    exact_path = (h.exact_path[0][::-1]).zfill(len(next(iter(dist_binary_prob))))
+    count1 = []
+    count2 = []
+    colors = []
+    count3 = copy.copy(dist_binary_prob)
+    count3.pop(exact_path)
+
+    for path, prob in dist_binary_prob.items():
+        count1.append(path)

```

Comment:
It seems like these two are not 'count' but a path_history and and step_probability or something like that. Could use clearer variable name here?